airframe-json/src/main/scala/wvlet/airframe/json/JSON.scala

```diff
@@ -119,7 +119,7 @@ object JSON extends LogSupport {
     def toJSON: String
   }
 
-  final case object JSONNull extends JSONValue {
+  case object JSONNull extends JSONValue {
```

Comment:
This is not a deprecation, but a warning. `case object` is implicitly `final`, so it is redundant.